Review of developer section (#2168)
MultiSafepay/docs: content/developer/iframes.md

hidden: false
---
 
An inline frame, or `<iframe>`, is an HTML document embedded inside another HTML document on a [website](/glossaries/multisafepay-glossary/#website). 
 

> Although MultiSafepay doesn't prohibit embedding a payment page as `<iframe>`, we don't recommend you to do so. Reasons include:

Reviewer: ```suggestion
Although MultiSafepay doesn't prohibit embedding a payment page as an `<iframe>`, we don't recommend it. This is because:
```

> - Some payment methods may not work if you use an `<iframe>` for privacy and security reasons. 

Reviewer: ```suggestion
- Some payment methods may not work if you use an `<iframe>`, for privacy and security reasons. 
```

> - Some banks use scripts that can't load in `<iframe>` elements.

Reviewer: ```suggestion
- Some banks use scripts that can't run within `<iframe>` elements.
```

> - Modern browsers' safety checks on `<iframe>` elements are very strict, which might block them.

Reviewer: ```suggestion
- Modern browsers' can block them due to stricter security checks.
```
